### DISCUSSION TITLE
Eclipse & Dbeaver setup errors lots & lots!

### DISCUSSION THREAD

**Discussion by snmag** [dbeaver/dbeaver#16357] [Q&A]
Installed Eclipse RCP 2021-12, JRE 11 (within DBeaver 22.0.1), DBeaver CE 22.0.3 from github, followed "https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse" step by step WITHOUT SUCCESS!

Same issue with JRE 17!! (Must be some incompatibility.)

Need to know the exact setup that will get me debugging and adding code.

Many many Thanx for your help!!

**Comment by LonwoLonwo** [comment]
You can try previous Eclipse version.
And what exactly problem do you have?

**Comment by ShadelessFox** [comment]
Hello @snmag,

JRE 17 is not supported, so you have to stick with JRE 11 for now.

Eclipse tends to fail dependency resolution and such, so you literally have to refresh (<kbd>F5</kbd>) and update (<kbd>Alt+F5</kbd>) the Maven project several times, along with restarting Eclipse itself.

Please let me know if these tricks helped.
